dev/io.openliberty.microprofile.health.3.1.internal/bnd.bnd

```diff
@@ -97,7 +95,6 @@ testsrc: test/src
 -testpath: \
         ../build.sharedResources/lib/junit/old/junit.jar;version=file, \
         com.ibm.ws.junit.extensions;version=latest, \
-        com.ibm.websphere.javaee.jsonp.1.0;version=latest, \
```

Comment:
This might be needed for a test so may not want to remove it.

Reply:
ok, added back in